db/re/item_db.conf

```diff
@@ -5307,7 +5307,7 @@ item_db: (
 	EquipLv: 40
 	Subtype: "W_1HSWORD"
 	Script: <"
-		bonus2 bSubRace,RC_NonBoss,-10;
+		bonus2(bAddRaceTolerance, RC_NonBoss, -10);
```

Comment:
in Aegis this is a class instead of race tolerance (and shall be fixed to class or left as is)